Makefile

```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
+UP_VERSION = v0.13.0
```

Comment:
I'm not sure what this is for

Reply:
Sets the version for https://github.com/upbound/up 👍🏻